docs/reference/query-languages/esql/functions-operators/conditional-functions-and-expressions.md

```diff
@@ -0,0 +1,27 @@
+---
+navigation_title: "conditional functions and expressions"
```

Comment:
would it be possible to make these [sentence case](https://docs.elastic.dev/tech-writing-guidelines/content-structure#topic-titles)?

Reply:
I followed the link, but it did not define 'sentence case', merely suggested using it. Perhaps you could provide a suggestion for an alternative navigation_title, so I know better what you mean here?